src/queries/user/collection/articles.ts

```diff
@@ -29,20 +29,40 @@ const resolver: GQLCollectionResolvers['articles'] = async (
     return connectionFromArray([], { first, after }, count)
   }
 
-  const [articles, totalCount] =
-    await collectionService.findAndCountArticlesInCollection(collectionId, {
-      skip,
-      take,
-      reversed,
-    })
-
-  return connectionFromPromisedArray(
-    atomService.articleIdLoader.loadMany(
-      articles.map(({ articleId }) => articleId)
-    ),
-    { first, after },
-    totalCount
-  )
+  const loadArticles = async (
+    articles: CollectionArticle[],
+    totalCount: number,
+    pageNumber: number | null = 1
+  ) => {
+    const connection = await connectionFromPromisedArray(
+      atomService.articleIdLoader.loadMany(
+        articles.map(({ articleId: aid }) => aid)
+      ),
+      { first, after },
+      totalCount
+    )
+
+    return pageNumber ? { ...connection, pageNumber } : connection
```

Comment:
if we return `pageNumber` in this resolver (`src/queries/user/collection/articles.ts`), you have to revise `ArticleConnection` (`src/types/collection.ts#L26`) so that client-side can query this field.